packages/components/src/custom-select-control-v2/types.ts

```diff
@@ -40,15 +33,131 @@ export type CustomSelectProps = {
 	/**
 	 * The size of the control.
 	 *
-	 * @default 'default'
+	 * @default default
```

Comment:
We've been using this convention for string values. The quotations marks look a bit redundant in Storybook prop tables but it's technically correct.

```suggestion
	 * @default 'default'
```